openwisp_users/tests/test_admin.py

```diff
@@ -311,9 +311,10 @@ def test_operator_change_organization(self):
         response = self.client.get(
             reverse(
                 f'admin:{self.app_label}_organization_change', args=[default_org.pk]
-            )
+            ),
+            follow=True,
```

Comment:
rather than following the redirect, assert the status code to ensure it's a redirect and remove the following lines which are not relevant anymore